### ISSUE TITLE
Question: How can I import functions? All i keep getting is references...

### ISSUE THREAD

**Issue by jsProj** [laverdet/isolated-vm#51]
```
const go = ($ = {}) => {
    const vm = $.vm || newVM({ memoryLimit: 8 });
    const globals = $.globals || {};
    const ctx = $.context || newContext(vm);
    const gl = globalSetter(ctx);
    Object.keys(globals).forEach(name => {
        const inp = globals[name];
        gl.setSync(`__${name}`, new ivm.Reference(inp));
    });
    // v This one was taken from example v
    gl.setSync("_log", new ivm.Reference(function (...$) {
        console.log(...$);
    }));
    // ^^^^^^^^^^
    let script = $.code || (() => { });
    vm.compileScriptSync(`new ` + script).runSync(ctx);
};
```

Not even the exampe code worked, it still comes up as a reference.

**Comment by laverdet** [comment]
This looks like a fragment of some larger piece of code.. I can't really make sense of what you're trying to do here because there are too many unknowns.

A `Reference` instance is a reference to some remote JS object that may or may not live in another isolate. To invoke a reference as a function you would call the `apply` or `applySync` method on that reference.

As far as I can tell all the example code in the README does in fact work correctly. The npm package is a bit out of date though.. For instance `context.global` was added recently to `master` but wasn't on npm. I just pushed 1.2.0 to npm which should better match the README.
